homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Also would be great to add some `margin-bottom` to the paragraphs if you reset all default indentation. Cause now they all are stacked together.
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/ca9d74e9-3a21-4b6b-b602-119abdf60f33)

Reply:
fix: add margin-top for text